ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/TipsLayout.kt

```diff
@@ -0,0 +1,96 @@
+package org.cru.godtools.tutorial
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+
+//import com.airbnb.lottie.compose.*
+//import org.cru.godtools.R
+import org.cru.godtools.base.ui.theme.GodToolsTheme
+import java.nio.file.Files.size
+
+@Preview(showBackground = true)
+@Composable
+internal fun TipsLayout() = GodToolsTheme() {
+    Column(
+        modifier = Modifier.fillMaxSize(),
+        //verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ){
+        Spacer(modifier = Modifier.height(100.dp))
```

Comment:
so, you can actually implement the vertical spacer by using a `.padding(top = 100.dp)` modifier on the column.